City-of-Helsinki/helsinki-paatokset
# PP-213: Typography overhaul

- Delete font overrides from custom theme
- Change font-weight variables to ones from HDBT theme

To test:
- Run `make drush-cr`
- Check that the fonts are loaded OK and that the font-weight - font pairing matches the sketches
